### ISSUE TITLE
Move/copy runtime-common to fellowship repo

### ISSUE THREAD

**Issue by gpestana** [polkadot-fellows/runtimes#59]
Some of the implementations under polkadot-sdk's `polkadot-runtime-common` crate are critical to Polkadot and Kusama (e.g. `fn era_payout`, which implements the inflation formula. This logic should probably be under the umbrella of the fellowship. Since the westend and rococo runtimes also need the runtime-common, another option is to keep a different runtime-common crate in each repo.

**Comment by bkchr** [comment]
Yeah this requires some kind of clean-up. We probably should start by moving out the crates from `runtime-common` in the Parity repos. Then having a copy of this crate should be fine.

**Comment by joepetrowski** [comment]
Should also include https://github.com/polkadot-fellows/runtimes/pull/109#discussion_r1420818182

**Comment by bkontur** [comment]
relates to: https://github.com/paritytech/polkadot-sdk/issues/3054
